bootstrap/kubeadm/docs/external-etcd.md

```diff
@@ -11,13 +11,32 @@ Before getting started you should be aware of the expectations that come with us
     * As an example, cross availability zone traffic can cost money on cloud providers. You don't have to deploy etcd
     across availability zones, but if you do please be aware of the costs.
 
-### Getting started
+## Getting started
 
 To use this, you will need to create an etcd cluster and generate an apiserver-etcd-client key/pair.
-[`etcdadm`](https://github.com/kubernetes-sigs/etcdadm) is a good way to get started if you'd like to test this
-behavior.
+This behaviour can be tested using [`kubeadm`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) and [`etcdadm`](https://github.com/kubernetes-sigs/etcdadm).
 
-Once you create an etcd cluster, you will want to base64 encode the `/etc/etcd/pki/apiserver-etcd-client.crt`,
+
+### Using kubeadm 
+Once you created the etcd cluster, CA certificates are required.
+If you already have a CA then the CA's `key` and `crt` must be copied to `/etc/kubernetes/pki/etcd/ca.crt` and `/etc/kubernetes/pki/etcd/ca.key`.
+
+If you do not already have a CA then run command `kubeadm init phase certs etcd-ca`. This creates two files
+
+ * `/etc/kubernetes/pki/etcd/ca.crt`
+ * `/etc/kubernetes/pki/etcd/ca.key`  
```

Comment:
It's not clear to me what this generated CA keypair is intended to be used for, especially since the `etcdadm` directions mention creating a separate CA keypair.

Reply:
Thanks for catching this! etcdadm creates the CA if one does not exist, uses it sign its server and peer certificates, and finally to sign the apiserver etcd client certificate.

@prankul88 Did you have any issues using etcdadm's CA with kubeadm?

Reply:
@dlipovetsky Did not use etcdadm's CA with kubeadm. Will check

Reply:
> It's not clear to me what this generated CA keypair is intended to be used for

@detiber  Would you like me to add more details to that?

Reply:
@prankul88 please do, with the current documentation it's not clear what the use of `kubeadm` is for here and which of the generated certificate or CA keypairs are intended to be used in the KubeadmConfig resources generated for the Cluster that will be consuming the external etcd.

Reply:
@detiber PTAL. Thanks